### DISCUSSION TITLE
Encountering 'ArtifactInputFailure' error code when attempting to Create Sample Workload Item

### DISCUSSION THREAD

**Discussion by pallar-ms** [microsoft/Microsoft-Fabric-workload-development-sample#47] [Help] [answered]
Followed the instructions for the dev setup for the Sample Workload - https://learn.microsoft.com/en-us/fabric/workload-development-kit/quickstart-sample?source=docs.
When attempting to create a Sample Workload Item, the n/w logs show the following request failing:
Request URL:
https://wabi-us-east-a-primary-redirect.analysis.windows.net/metadata/workspaces/*workspaceId*/artifacts
Request Method:
POST
Status Code:
400 Bad Request

{
    "error": {
        "code": "ArtifactInputFailure",
        "pbi.error": {
            "code": "ArtifactInputFailure",
            "parameters": {
                "FieldName": null,
                "FailureReason": null
            },
            "details": [],
            "exceptionCulprit": 1
        }
    }
}

There were no changes made to the workload payload in the sample code and the contracts look right, so it is unclear why the payload is resulting into a BadRequest.

**Comment by ramizsha** [comment, marked as answer]
@pallar-ms things to double check:
1. Your devgateway succeeded to register your dev instance.
2. Your backend server is running.
3. You are trying to create an item on a workspace that is assigned to the same capacity that the dev instance was registered on.

**Reply by pallar-ms** [reply]
Looks like #3 was the issue for me. The workspace I had created was assigned to the wrong capacity (default settings). After reassigning it to the right one, it worked. Thanks!
